### ISSUE TITLE
Fix FP S1128 (`unused-import`): support lowercase tag name

### ISSUE THREAD

**Issue by ilia-kebets-sonarsource** [SonarSource/SonarJS#3973]
When using VueJS, elements with tags are considered as used if they [start with an uppercase](https://github.com/SonarSource/SonarJS/blob/9c622894c47db11d3483caf38ae02df502b90b06/src/linting/eslint/rules/unused-import.ts#L210) or if they [contain a dash](https://github.com/SonarSource/SonarJS/blob/9c622894c47db11d3483caf38ae02df502b90b06/src/linting/eslint/rules/unused-import.ts#L212). It does not take into account when starting with a small letter, or in camel case.

This issue was raised on community: https://community.sonarsource.com/t/fp-s1128-false-positive-with-vue-js-imports/91778

### Naive solution proposal

Compare the strings in a case insensitive way, removing dashes like:

```typescript
function compare(tag: string, imported: string) {
  return sanitize(tag) === sanitize(imported);

  function sanitize(name: string) {
    const withoutDashes = name.split('-').join('');
    return withoutDashes.toLowerCase();
  }
}
```

There is a risk of FNs though!

**Comment by alexander-kamushkin-sonarsource** [comment]
Should be fixed with https://github.com/SonarSource/SonarJS/pull/4083

**Comment by ferferga** [comment]
Thank you very much @alexander-kamushkin-sonarsource !
